decidim-core/app/cells/decidim/card_m_cell.rb

```diff
@@ -58,18 +58,6 @@ def decidim
       Decidim::Core::Engine.routes.url_helpers
     end
 
-    def has_author?
```

Comment:
I think we should keep the logic here, so each card can overwrite how to render their author if needed

https://github.com/decidim/decidim/pull/3727#discussion_r198733600

Reply:
Cells are going complicated and what you propose is breaking encapsulation. Why a parent card should decide how authorships should be rendered if we already have cells to tackle this visualization details?

If we want a coherent UI we need to promote homogeneization, not the contrary, don't you think?
Also YAGNI, if some card need to overwrite this authorship cell for some reason, it will be better to tackle the problem when we know the whole information. In any case, we can extract the call to the coauthorships cell into a method so that it can be overwritten with specific logic.

Reply:
Ok, then, but please check that no other card is overwriting the `author.erb` view then 😄

Reply:
I guess I misunderstood your first comment! I first thought you meant to keep the logic of selecting what to render (if author cell or coauthorship cell) but I think you meant to have a way to select if authors should be rendered or not.

Then I think you are right, to keep a has_authors? method to flag if authorship should be rendered or not. Is this what you meant?

Reply:
No, you understood right! 😄 But you made a good point, so 👍 for the change.

Still, what you propose (the `has_authors?` flag) looks very useful, so 👍 on adding it!